backends/aerospike.go

```diff
@@ -3,69 +3,123 @@ package backends
 import (
 	"context"
 	"errors"
+	"fmt"
 
 	as "github.com/aerospike/aerospike-client-go"
+	ase "github.com/aerospike/aerospike-client-go/types"
 	"github.com/prebid/prebid-cache/config"
 	"github.com/prebid/prebid-cache/metrics"
 	log "github.com/sirupsen/logrus"
 )
 
-const setName = "uuid"
-const binValue = "value"
+const SET_NAME = "uuid"
```

Comment:
Should it be `SetName` and `BinValue` if these constants are exported, otherwise `setName` and `binValue` to correspond to go conventions?

Reply:
Done